opentreemap/treemap/templates/treemap/partials/search_location_polygon_editing.html

```diff
@@ -22,15 +22,16 @@
             {% trans "CUSTOM AREA" %}
         </span>
         &nbsp;
-        <a class="js-edit-area">
+        <a class="edit-area">
             <i class="icon-pencil"></i>
         </a>
     </div>
-    <div class="location-search-alternate" id="edit-area-controls">
-        <span class="text">
-            {% trans "Edit area" %}
-            <a class="js-edit-area-done" style="font-weight:bold;">DONE</a>
-            <a class="js-edit-area-cancel">CANCEL</a>
-        </span>
+    <div class="location-search-alternate" id="edit-area-controls"
+         style="text-align:justify">
+        <span class="text">{% trans "Edit area" %}</span>
+        <div class="controls text">
+            <a class="save-area">DONE</a>&nbsp;&nbsp;&nbsp;&nbsp;
+            <a class="cancel-edit" style="font-weight:normal;">CANCEL</a>
```

Comment:
IMO it's simpler to put one-off styles like this in the markup rather than making the user search for it in a big SASS file and (maybe) discover that it's a one-off.